build.ps1

```diff
@@ -233,9 +250,6 @@ function CreateZips([string] $runtimeSuffix) {
 function deleteDuplicateWorkers() {
     Write-Host "Deleting workers directory: $privateSiteExtensionPath\32bit\workers" 
     Remove-Item -Recurse -Force "$privateSiteExtensionPath\32bit\workers" -ErrorAction SilentlyContinue
-    Write-Host "Moving workers directory:$privateSiteExtensionPath\64bit\workers to" $privateSiteExtensionPath 
-    
-    Move-Item -Path "$privateSiteExtensionPath\64bit\workers"  -Destination "$privateSiteExtensionPath\workers" 
```

Comment:
We should check with @pragnagopa on this -- I think this was a recent change that introduced this new error.

Reply:
Checked with @pragnagopa on this one and confirmed this was removed for building site extension.